Dockerfile.armhf

```diff
@@ -0,0 +1,20 @@
+FROM resin/armhf-alpine:3.5
```

Comment:
The reason to use `resin/armhf` since it bundles qemu so that it can be built on travis.